rawpy/_rawpy.pyx

```diff
@@ -704,11 +705,27 @@ cdef class RawPy:
     property white_level:
         """
         Level at which the raw pixel value is considered to be saturated.
+        This value is calculated from the data for most cameras, and hardcoded for others.
```

Comment:
As mentioned, I don't think this is an accurate enough explanation of what's going on with LibRaw's colordata.maximum

Reply:
How so? Should I elaborate more on its potential shortcomings (adding notes on `camera_white_level_per_channel`)? I wouldn't want to deviate too much from LibRaw's own docs, as the wrapper library should remain true to the underlying lib.